<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Diego Iastrubni &lt;diegoiast@gmail.com&gt;</author>
 <class>FileSystemBrowser</class>
 <widget class="QWidget" name="FileSystemBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>176</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin" stdset="0">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin" stdset="0">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="backButton">
       <property name="toolTip">
        <string>Back</string>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/trolltech/styles/commonstyle/images/left-32.png</normaloff>:/trolltech/styles/commonstyle/images/left-32.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="upButton">
       <property name="toolTip">
        <string>Go one directory up</string>
       </property>
       <property name="text">
        <string>Up</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/trolltech/styles/commonstyle/images/parentdir-32.png</normaloff>:/trolltech/styles/commonstyle/images/parentdir-32.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="homeButton">
       <property name="toolTip">
        <string>Reload</string>
       </property>
       <property name="text">
        <string>Home</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/trolltech/styles/commonstyle/images/computer-32.png</normaloff>:/trolltech/styles/commonstyle/images/computer-32.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="forwardButton">
       <property name="toolTip">
        <string>Forward</string>
       </property>
       <property name="text">
        <string>Forward</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/trolltech/styles/commonstyle/images/right-32.png</normaloff>:/trolltech/styles/commonstyle/images/right-32.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="locationLineEdit"/>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QListView" name="listView"/>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin" stdset="0">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>5</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Filter</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>5</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="filterEdit">
       <property name="text">
        <string>*.*</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="0" margin="0"/>
 <resources/>
 <connections/>
</ui>
